app/src/main/java/com/animal_villa/ui/theme/Color.kt

```diff
@@ -1,6 +1,6 @@
 package com.animal_villa.ui.theme
+import androidx.compose.ui.graphics.Color
 
-import android.graphics.Color
 
 //Change Colors Later
-val pink = Color()
\ No newline at end of file
+val pink = Color(0xFFFFC0CB)
```

Comment:
Used compose ui to use the pink color which was assigned earlier https://developer.android.com/jetpack/androidx/releases/compose-ui